test/functional/wallet_multisig_descriptor_psbt.py

```diff
@@ -69,7 +70,7 @@ def participants_create_multisigs(self, external_xpubs, internal_xpubs):
                     "timestamp": "now",
                 },
             ])
-            assert all(r["success"] for r in result)
+            assert_true(all(r["success"] for r in result))
```

Comment:
Can we unify the remaining `assert all()` constructs as well, e.g. https://github.com/bitcoin/bitcoin/blob/master/test/functional/wallet_conflicts.py#L152 ?